pandas/_libs/tslibs/period.pyx

```diff
@@ -2218,15 +2215,15 @@ cdef class _Period:
     def __repr__(self) -> str:
         base, mult = get_freq_code(self.freq)
         formatted = period_format(self.ordinal, base)
-        return "Period('%s', '%s')" % (formatted, self.freqstr)
+        return f"Period('{formatted}', '{self.freqstr}')"
 
     def __str__(self) -> str:
         """
         Return a string representation for a particular DataFrame
         """
         base, mult = get_freq_code(self.freq)
         formatted = period_format(self.ordinal, base)
-        value = ("%s" % formatted)
+        value = f"{formatted}"
```

Comment:
this is just `str(formatted)`?

Reply:
> this is just `str(formatted)`?

Absolutely correct, I get to auto mode after a few changes. haven't saw that